index.ts

```diff
@@ -116,7 +116,7 @@ const ERROR = {
 Forgot ${PROJECT_NAME} commands? Get help:\n  ${PROJECT_NAME} --help`,
   CONFLICTING_FILE_EXTENSION: (name: string) => `File names: ${name}.js/${name}.gs conflict. Only keep one.`,
   CREATE: 'Error creating script.',
-  DEPLOYMENT_COUNT: `Unable to deploy; Only one deployment can be created at a time`,
+  DEPLOYMENT_COUNT: `Unable to deploy; Scripts may only have up to 20 versioned deployments at a time.`,
```

Comment:
I could not get the previous error message, but I often got this. If you think they should be 2 separate messages, I can change that.